### PR TITLE
Java 6390

### PR DESCRIPTION
**Pull request by freelansam**

### CODE REVIEW

File: apache-kafka/pom.xml [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-streams</artifactId>
+            <version>${kafka.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${slf4j.version}</version>
```
Comment (kwoyke) [inline apache-kafka/pom.xml:29]: Let's use org.slf4j.version property from the parent pom.xml.

File: apache-kafka/src/main/java/com/baeldung/flink/model/InputMessage.java [hunk 2]
```diff
@@ -55,13 +56,12 @@ public InputMessage(String sender, String recipient, LocalDateTime sentAt, Strin
...
+        if (this == o)
+            return true;
+        if (o == null || getClass() != o.getClass())
+            return false;
         InputMessage message1 = (InputMessage) o;
-        return Objects.equal(sender, message1.sender) &&
-                Objects.equal(recipient, message1.recipient) &&
-                Objects.equal(sentAt, message1.sentAt) &&
-                Objects.equal(message, message1.message);
+        return Objects.equal(sender, message1.sender) && Objects.equal(recipient, message1.recipient) && Objects.equal(sentAt, message1.sentAt) && Objects.equal(message, message1.message);
```
Comment (kwoyke) [inline apache-kafka/src/main/java/com/baeldung/flink/model/InputMessage.java:64]: Let's keep the original formatting - each Objects.equal in its own line.

File: apache-kafka/pom.xml (hunk 1 above)
Comment (freelansam) [reply apache-kafka/pom.xml]: Done!

File: apache-kafka/src/main/java/com/baeldung/flink/model/InputMessage.java (hunk 2 above)
Comment (freelansam) [reply apache-kafka/src/main/java/com/baeldung/flink/model/InputMessage.java]: Done, please check!

**Pull request merged**